src/ngAnimate/module.js

```diff
@@ -267,9 +267,22 @@
  * .message.ng-enter-prepare {
  *   opacity: 0;
  * }
- *
  * ```
  *
+ * ### Animating between subsequent value changes
+ *
+ * Sometimes you need to animate between multiple (2 and more), different expression states, whose values
+ * don't necessary need to be known or referenced in CSS styles.
+ * Unless possible with more proper ["animation aware" directive](#directive-support), that specific
+ * use case can always be covered with {@link ngAnimate.directive:ngAnimateSwap} as can be seen in
+ * {@link ngAnimate.directive:ngAnimateSwap#examples this example}.
+ *
+ * Note that {@link ngAnimate.directive:ngAnimateSwap} is a *structural directive*, which means it
+ * creates a new instance of the element (including any other/child directives it may have) and
+ * links it to a new scope every time *swap* happens. In some cases this might not be desirable
+ * (e.g. for performance reasons, or when wish to retain internal state on the original
+ * element's instance).
```

Comment:
_element's instance_ --> _element instance_ (maybe?)

Reply:
It seems my english teacher was too much into adding `'s` always when you want to write about [possession](https://dictionary.cambridge.org/grammar/british-grammar/determiners/possession-john-s-car-a-friend-of-mine) :D

Reply:
Well, I'm not a native speaker either, so I could be wrong (😱).
But, I don't think it is about possesion here (the instance does not _belong_ to the element).